src/assets/scss/style.scss

```diff
@@ -6,3 +6,6 @@ body {
   font-size: 1.6rem;
```

Comment:
why do you still have two css files , may you kindly remove one you are not using .

Reply:
resolved